tf-psa-crypto/programs/demo_common.sh

```diff
@@ -0,0 +1,137 @@
+## Common shell functions used by demo scripts programs/*/*.sh.
```

Comment:
Better not to duplicate this file. For the time being, please just use the `programs/demo_common.sh` one even for PSA programs in `tf-psa-crypto/programs` (some adaptation needed maybe). Eventually we should move demo_common.sh to the framework and adapt run_demos.py and demo_common.sh accordingly. We will do that rather as part of the migration of the all.sh crypto components to tf-psa-crypto.

Reply:
Resolved now, just requiring a rebase.